data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -177,6 +177,7 @@ podman run -d --net host --privileged --name ironic-inspector \
      --restart on-failure \
      --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
      --env HTTP_BASIC_HTPASSWD=${IRONIC_HTPASSWD} \
+     --entrypoint /bin/runironic-inspector \
```

Comment:
I think we also need to replace all `IRONIC_INSPECTOR_IMAGE` references with `IRONIC_IMAGE` right?

I still see the inspector image in the release payload, but it seems likely we'll want to remove that when this is fixed to use the combined ironic-image?

Reply:
yup, will do

Reply:
lgtm, could you just update the commit message to reflect the change, e.g move to using the ironic-image which now also contains the inspector dependencies and entrypoint?